spanner/session_test.go

```diff
@@ -1574,12 +1574,20 @@ func TestMaintainer(t *testing.T) {
 			t.Fatalf("cannot get session from session pool: %v", err)
 		}
 	}
-	sp.mu.Lock()
-	g, w := sp.numOpened, sp.MinOpened+sp.incStep
-	sp.mu.Unlock()
-	if g != w {
-		t.Fatalf("numOpened sessions mismatch\nGot: %d\nWant: %d", g, w)
-	}
+	// It is possible that we need to wait a little before all sessions have been created.
+	// We requested 20 sessions, but the default step that BatchCreateSessions will use is
+	// 25 sessions. It is therefore possible that 20 sessions have been returned to the
+	// test, without all the sessions that have been requested from the server have been
```

Comment:
nits: consider to rephrase the last sentence. 

> It is therefore possible that 20 sessions have been returned to the test, without all the sessions that have been requested from the server have been added to the pool.

I think this means that when we execute the check, some sessions are still not returned to the pool (< 5 + 25) yet so that we need to wait for a bit.

Reply:
Yep, that comment was not really clarifying very much... I've changed it into a better description of what is actually happening.